pkg/agents/team7/agent1/agent.go

```diff
@@ -0,0 +1,188 @@
+package team7agent1
+
+import (
+	"github.com/SOMAS2021/SOMAS2021/pkg/infra"
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/globalTypes/food"
+	"math/rand"
+	"time"
+)
+
+/*
+
+Openness 	high-> doesn't get effected by new floors, looks forward to new day
+			low-> does get effected by new floors
+
+Currently implemented by changing attributes on reshuffles
+
+Conscientiousness high-> plans ahead, attentive and takes into account messgages
+			low-> no planning, fails to complete assigned tasks
+
+Not implemented yet
+
+
+Extraversion high-> very likely to communicate, will share a lot of information
+			low-> does not like to communicate
+
+Not implemented yet
+
+Agreeableness high-> caring so more likely to go hungry for the betterment of others, trustworthy
+			low-> greedy, manipulative
+
+
+Implemented by intialising attributes, not considering health state yet
+
+
+Neuroticism high-> dramatic mood swings, struggles to recover after weak state
+			low-> stable, relaxed and resilient
+
+
+
+
+*/
+
+type team7Personalities struct {
+	openness int
+
+	conscientiousness int
+
+	extraversion int
+
+	agreeableness int
+
+	neuroticism int
+}
+
+type CustomAgent7 struct {
+	*infra.Base
+	personality team7Personalities
+	greediness  int
+	kindness    int
+	daysAlive   int
```

Comment:
```suggestion
```

Reply:
This was added as a field to base agent as of last night.

Reply:
using AGe